STANDARD.md

```diff
@@ -661,6 +662,7 @@ Reminder: for scalar records the `record` itself is also the `component`.
 
   - `timeOffset`
     - type: *(floatX)*
+    - scope: *optional* (only used if `time` and `dt` are used)
```

Comment:
Maybe let's move this from *Required for each `Record`* to a section optional (or *recommended* if we decide so).